src/components/Container.tsx

```diff
@@ -1,24 +1,41 @@
 import React, { ReactNode } from 'react'
-import { Platform, Dimensions, StyleSheet, View, ViewProps } from 'react-native'
+import {
+  StyleSheet,
+  View,
+  SafeAreaView,
+  ViewProps,
+  ColorValue,
+} from 'react-native'
 
-const { height: deviceHeight } = Dimensions.get( 'window' )
+import COLORS from '../themes/colors'
 
 const styles = StyleSheet.create( {
   main: {
     flex: 1,
-    height: Platform.OS === 'ios' ? deviceHeight : deviceHeight - 20,
-    backgroundColor: 'white',
+    backgroundColor: COLORS.White,
   },
 } )
 
 type ContainerProps = {
   children: ReactNode,
+  /**
+   * Color value for Status Bar
+   */
+  statusBarColor?: ColorValue,
 } & ViewProps
 
-const Container = ( { children, ...props }: ContainerProps ) => (
-  <View style={styles.main} {...props}>
-    {children}
-  </View>
+const Container = ( {
+  children,
+  statusBarColor = COLORS.White,
+  style,
+  ...props
+}: ContainerProps ) => (
+  <>
+    <SafeAreaView style={{ flex: 0, backgroundColor: statusBarColor }} />
```

Comment:
Why not nest the following view inside of this?

Also, no inline styles please